inst/include/population_dynamics/population/population.hpp

```diff
@@ -457,7 +457,7 @@ struct Population : public fims_model_object::FIMSObject<Type> {
                  this->weight_at_age[age];
       } else {
         POPULATION_LOG << "fleet " << fleet_ << " is a survey" << std::endl;
-        index_ = this->fleets[fleet_]->q *
+        index_ = this->fleets[fleet_]->q[0] *
```

Comment:
This q[0] doesn't work if we have an annual q vector. In fleet.hpp we should change the default to resize to length years so that this can always index over years (or whatever dimensionality we are allowing variation over)

Reply:
fixed